includes/Story_Post_Type.php

```diff
@@ -685,6 +683,44 @@ public static function filter_template_include( $template ) {
 		return $template;
 	}
 
+	/**
+	 * Gets a valid publisher logo URL. Loops through sizes and looks for a square image.
+	 *
+	 * @param {int} $image_id Attachment ID.
+	 *
+	 * @return string|false Either the URL or false if error.
+	 */
+	private static function get_valid_publisher_image( $image_id ) {
+		$logo_image_url = false;
```

Comment:
There this is a fallback? Like the WP logo

Reply:
I'm not certain actually if there should be a fallback if the publisher logo is set. If the logo is set but it's not correct then the editor should notify about it, since it's marked as `required`.

Currently, the fallback is used only if the publisher logo is not set.

IDK what's the ideal behavior here until the editor enforces the correct size image. Thoughts?

Reply:
OK, always adding a fallback for now, this should be changed later though.